### ISSUE TITLE
Error while resuming restore of emails

### ISSUE THREAD

**Issue by chirag-jog** [gaubert/gmvault#214]
When i try to resume the restoration of my emails using the following command:
`./gmvault restore xxx.yyy@gmail.com --resume --apply-label "zzz-email"`

I get the following python traceback. I have tried this several times over a few days and all attempts to restore fail. Any insights would be useful here.

```
Jul 25 16:03:42 MacBook-Pro.local gmv_runner[35870] <Notice>: Get OAuth2 credential from /Users/cheezo/.gmvault/chirag.jog@gmail.com.oauth2.
Jul 25 16:03:42 MacBook-Pro.local gmv_runner[35870] <Notice>:
Jul 25 16:03:45 MacBook-Pro.local gmv_runner[35870] <Notice>: Error when reaching Gmail server. Wait 4 second(s) and retry up to 2 times.
Jul 25 16:03:45 MacBook-Pro.local gmv_runner[35870] <Notice>: Error: APPEND command error: BAD ['Could not parse command'].
Jul 25 16:03:45 MacBook-Pro.local gmv_runner[35870] <Notice>:
Jul 25 16:03:45 MacBook-Pro.local gmv_runner[35870] <Notice>: === Exception traceback ===
Jul 25 16:03:45 MacBook-Pro.local gmv_runner[35870] <Notice>: Traceback (most recent call last):
Jul 25 16:03:45 MacBook-Pro.local gmv_runner[35870] <Notice>:   File "gmv/gmv_cmd.pyc", line 745, in run
Jul 25 16:03:45 MacBook-Pro.local gmv_runner[35870] <Notice>:   File "gmv/gmv_cmd.pyc", line 621, in _restore
Jul 25 16:03:45 MacBook-Pro.local gmv_runner[35870] <Notice>:   File "gmv/gmvault.pyc", line 882, in restore
Jul 25 16:03:45 MacBook-Pro.local gmv_runner[35870] <Notice>:   File "gmv/gmvault.pyc", line 1119, in restore_emails
Jul 25 16:03:45 MacBook-Pro.local gmv_runner[35870] <Notice>:   File "gmv/gmvault.pyc", line 69, in handle_restore_imap_error
Jul 25 16:03:45 MacBook-Pro.local gmv_runner[35870] <Notice>: error: APPEND command error: BAD ['Could not parse command']
Jul 25 16:03:45 MacBook-Pro.local gmv_runner[35870] <Notice>:
Jul 25 16:03:45 MacBook-Pro.local gmv_runner[35870] <Notice>: === End of Exception traceback ===
```

**Comment by gaubert** [comment]
@chirag-jog  What version are you using and on which platform ?
I need to get more info to help. Please re-run gmvault with the --debug option and send me via email the complete console output such that I can understand the source of the issue.

Thanks.

**Comment by gaubert** [comment]
@chirag-jog fixed in gmv-1.9.1 see #218

**Comment by gaubert** [comment]
close it now
